src/trivia/util.h

```diff
@@ -42,6 +42,8 @@
 #include <stdbool.h>
 
 #if defined(__cplusplus)
+#include <stdalign.h>
+
```

Comment:
Please move it closer to the alignas definition so that it's clear what it is needed for:

```diff
diff --git a/src/trivia/util.h b/src/trivia/util.h
index da640c964273..3d853f612968 100644
--- a/src/trivia/util.h
+++ b/src/trivia/util.h
@@ -233,6 +233,9 @@ strnindex(const char **haystack, const char *needle, uint32_t len, uint32_t hmax
  *
  * \example struct obuf { int a; int b; alignas(16) int c; };
  */
+#if defined(__cplusplus)
+#  include <stdalign.h>
+#endif
 #if !defined(alignas) && !defined(__alignas_is_defined)
 #  if __has_feature(c_alignas) || (defined(__GNUC__) && __GNUC__ >= 5)
 #    include <stdalign.h>
```

Reply:
Done